src/Components/MailtoDropdown.js

```diff
@@ -49,10 +127,13 @@ export default class MailtoDropdown extends preact.Component {
         };
         const handler_buttons = handlers.map(h => (
             <a
-                href={MAILTO_HANDLERS[h].link(data)}
+                href={MAILTO_HANDLERS[h].link && MAILTO_HANDLERS[h].link(data)}
```

Comment:
```suggestion
                href={MAILTO_HANDLERS[h].link?(data)}
```
After rebasing that can be simplified.

Reply:
Actually, that's `href={MAILTO_HANDLERS[h].link.?(data)}` as per [1](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining.html#calling-deeply-nested-functions). :)